packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -406,7 +406,8 @@ class C4DFooterComposite extends MediaQueryMixin(
 
         <c4d-legal-nav
           size="${ifDefined(size)}"
-          navLabel="${ifDefined(navLabel)}">
+          navLabel="${ifDefined(navLabel)}"
+          adjunctLinksLenght="${adjunctLinks.length ?? 0}">
```

Comment:
There is a typo here, should be `adjunctLinksLength`. Looks like the typo carries to other places its being used here as well.

Reply:
Thanks for the review. 
That was my bad. 
It is fixed in my latest commit.